openslides/users/static/templates/users/user-import.html

```diff
@@ -23,81 +23,148 @@ <h3 translate>Import by copy/paste</h3>
+        class="btn btn-default">
+      <i class="fa fa-angle-double-left fa-lg"></i>
+      <translate>Back to users overview</translate>
+    </a>
+  </div>
 
   <hr>
 
-  <h3 translate>Import by CSV file</h3>
-  <p translate>Select a CSV file to import users!
+  <h2 translate>Import by CSV file</h2>
 
-  <p translate>Please note:</p>
+  <div class="block row">
+    <div class="title">
+      <h3 translate>Select a CSV file
+    </div>
+    <div class="block right import">
+      <label class="label" for="inputSeparator" translate>Separator</label>
+      <input type="text" ng-model="separator" ng-change="setSeparator()" ng-init="separator=separator" id="inputSeparator">
+      <br>
+      <label class="label" for="selectEncoding" translate>Encoding</label>
+      <select ng-model="encoding" ng-options="enc as enc for enc in encodingOptions"
+          ng-selected="setEncoding()" ng-init="encoding=encoding" id="selectEncoding"></select>
+      <ng-csv-import
+          content="csv.content"
+          header="csv.header"
+          header-visible="csv.headerVisible"
+          separator="csv.separator"
+          separator-visible="csv.separatorVisible"
+          result="csv.result"
+          encoding="csv.encoding"
+          encoding-visible="csv.encodingVisible"></ng-csv-import>
+    </div>
+  </div>
+
+  <h4 translate>Please note:</h4>
   <ul>
-      <li><translate>Required comma separated values</translate>:<br>
-          <code translate>'title, first_name, last_name, structure level, groups, comment, is active'</code>
+      <li><translate>Required comma or semicolon separated values with these column header names in the first row</translate>:<br>
+          <code>'title, first_name, last_name, structure level, groups, comment, is active'</code>
       <li><translate>Default groups</translate>:
           <translate>Delegate</translate> <code>3</code>,
           <translate>Staff</translate> <code>4</code>
       <li translate>At least first name or last name have to be filled in. All
         other fields are optional and may be empty.
-      <li translate>The header in first line is required.
-      <li translate>Required CSV file encoding is UTF-8.
+      <li translate>Only double quotes are accepted as text delimiter (no single quotes).
       <li><a href="https://github.com/OpenSlides/OpenSlides/wiki/CSV-Import" translate>
             Use the CSV example file from OpenSlides Wiki.
           </a>
   </ul>
 
-  <ng-csv-import
-    content="csv.content"
-    class="import"
-    header="csv.header"
-    separator="csv.separator"
-    result="csv.result"></ng-csv-import>
-
-  <div ng-if="csv.result" class="col-sm-10 well">
+  <div ng-if="csv.result">
     <h3 translate>Preview</h3>
     <table class="table table-striped table-bordered table-condensed">
       <thead>
         <tr>
+          <th>
           <th>#
           <th translate>Title
```

Comment:
This line and the next lines do not have closing tags. Are they needed for translation?